x-pack/solutions/observability/plugins/infra/public/components/asset_details/hooks/use_host_metrics_charts.ts

```diff
@@ -31,30 +37,38 @@ export const useHostCharts = ({
         },
       }),
     }));
-  }, [dataViewId, metric, overview]);
+  }, [dataViewId, metric, overview, schema]);
 
   return { charts, error };
 };
 
 export const useKubernetesCharts = ({
   dataViewId,
   overview,
+  schema,
 }: {
   dataViewId?: string;
   overview?: boolean;
+  schema?: DataSchemaFormat | null;
 }) => {
-  const model = useMemo(() => findInventoryModel('host'), []);
+  const model = findInventoryModel('host');
 
   const { value: charts = [], error } = useAsync(async () => {
-    const { kibernetesNode } = await model.metrics.getCharts();
+    const { kubernetesNode } = await model.metrics.getCharts({
+      schema: schema ?? DataSchemaFormat.ECS,
+    });
+
+    if (!kubernetesNode) {
+      return [];
+    }
 
     const items = overview
-      ? [kibernetesNode.xy.nodeCpuCapacity, kibernetesNode.xy.nodeMemoryCapacity]
+      ? [kubernetesNode.xy.nodeCpuCapacity, kubernetesNode.xy.nodeMemoryCapacity]
```

Comment:
Omg, good catch, how long have we been with this typo? 😄

Reply:
Thanks to Windsurf 😆 . It would've passed unnoticed again if it weren't for it.